src/main/webapp/app/pages/companyPage/CompanyPage.tsx

```diff
@@ -218,6 +232,38 @@ export default class CompanyPage extends React.Component<
       });
   }
 
+  @autobind
+  onConfirmDeleteAccountButton() {
+    this.showLicenseChangeModal = false;
```

Comment:
Do me a favor to change the variant name? This does not reflect what it does anymore. Thanks!